src/foremast/utils/awslambda.py

```diff
@@ -33,6 +33,32 @@ def get_lambda_arn(app, account, region):
                                                                                                      region))
 
 
+def get_lambda_alias_arn(app, account, region):
```

Comment:
I'm not seeing a test case around this method and its obviously used in several locations.